lib/vagrant/ui.rb

```diff
@@ -103,12 +103,6 @@ def ask(*args)
         raise Errors::UIExpectsTTY
       end
 
-      [:detail, :warn, :error, :info, :output, :success].each do |method|
```

Comment:
Removing this is premature, I know. Several other commands still output `@ui.info` only, which would leave them outputting nothing in machine-readable mode. Hopefully once those commands have been updated we can remove this code block.

In the meantime, I was hoping to add a `--suppress-ui` flag, which would output only `@ui.machine` calls in that mode, to clean up and declutter the output. However, I am unsure of how to read global arguments in the `@ui` library. I can see how to do it in plugins, but not the library.

This is the very first Ruby I've ever written, so please bear with me if I'm a bit slow :-)

Reply:
:-1: on removing these methods. This is mainly due to the existing behavior of the UI class and is not backwards compatible. I'd also argue that it is trivial to filter out `ui` type messages if they are considered extraneous for the use case.

Reply:
Methods have been restored.